### ISSUE TITLE
homosapiens genebot does not function due to a missing key "src-version" in mygene.info

### ISSUE THREAD

**Issue by andrawaag** [SuLab/scheduled-bots#52]
genebot_homosapiens does not work. It reports a missing [src_version](http://jenkins.sulab.org/job/GeneBot_Homo_sapiens/89/console)

GeneBot.py On [line 858](https://github.com/SuLab/scheduled-bots/blob/master/scheduled_bots/geneprotein/GeneBot.py#L858) expects a value in ["src"]["src_version"] that does not seems to exist.
 
["src"] in [https://mygene.info/v3/metadata](https://mygene.info/v3/metadata) indeed does not have ["src-version"]
Has this recently been replaced and if so by what?

**Comment by andrewsu** [comment]
Looks like this issue is related to this older issue https://github.com/SuLab/scheduled-bots/issues/36.  It looks like in the commit that addressed that previous issue (https://github.com/SuLab/scheduled-bots/commit/edc952666eb0c55cb7bf40a1ba4423f6535bf9fb), the `src_version` info was simply moved to a different location in the json object.  However, as described in this blog post from around that same time https://mygene.info/blog/data-release-updates-and-metadata-deprecations the intent was the completely remove that node in favor of versions that are nested within each source, e.g.:
```
"src": {
    "PantherDB": {
      "code": {
        "branch": "v3",
        "commit": "2a4aeca",
        "folder": "src/plugins/PantherDB",
        "repo": "https://github.com/biothings/mygene.info.git",
        "url":  "https://github.com/biothings/..."
      },
      "stats": {
        "PantherDB": 156054
      },
      "version": "2017-12-11"
 }
```

Anyway, bottom line, it looks like we need to refactor all the bots that depend on that `src_version` to extract the info out of new object structure...

**Comment by andrawaag** [comment]
As per [run #91](http://jenkins.sulab.org/job/GeneBot_Homo_sapiens/91/console) on jenkins the issue is fixed with the above commits
